privacy-profile-composer/pkg/composer/grpc_client.go

```diff
@@ -75,3 +76,50 @@ func Run_client() {
 	}
 	log.Println(profile)
 }
+
+func sendComposedProfile(fqdn string, purpose string, piiTypes []string, thirdParties []string) api.StatusType {
```

Comment:
where is this function being used? there is an argument that if it returns `api.StatusType`, it will only be used within golang filter, so it should potentially live in `envoyfilter` package. what do you think :thinking:?

Reply:
This function is supposed to be called within the jaeger trace querying implementation, to submit updates to all observed profiles, after going through a batch of traces. We were planning to call it from the Golang filter (e.g. `DecodeBody` method), but we don't need to do that as we propagate info from the data plane to the control plane through the span tags. I added a todo to use / modify it whenever I'm working on the trace querying API. I also fixed the return type so it doesn't return `api.StatusType`.